### ISSUE TITLE
Proposed Interest Type: loans-thinly-capitalised

### ISSUE THREAD

**Issue by timgdavies** [openownership/data-standard#270]
From alca.ai:

> In the list of controls, maybe it is worth adding a new code: “loansThinlyCapitalised”.
> 
> Take an example with a capital of USD5,000, which obtains funding of USD 10 million. In practice the lender to the company is the real beneficial owner and the company just a shell. This is one of the standard mechanism of controlling companies without having in theory control, which may be worth spelling it out through a separate category

Views on this are welcome.

**Comment by ScatteredInk** [comment]
Thanks, this is useful for thinking about BODS in investigative contexts.
 
I'm unclear about whether this is about:

- _control_ (the loan gives the lender de facto control over the entity);
- _financial benefit_ (the loan is given to the entity for a financial purpose, though possibly for non-commercial reasons);
- some combination of control and financial benefit. 

If it is solely about _control_ then the suggested code is very specific but we might want to look at two alternative codes: `formal-control-granted-by-provision-of-finance` and `informal-control-granted-by-provision-of-finance`. The first could cover cases where loan or convertible funding has documented control provisions. The second would cover de facto control.

If it is about financial benefit then I am less convinced becuase this feels more like a company-level flag (`thinly-capitalized`) and potentially out-of-scope - the kind of thing that BO data allows you to do but shouldn't necessarily do itself. But I think we need to know more about this and I'll reach out to alca.ai to find out.

If it's a combination, then we would want to separate the two benefits. 

A couple of points on this.

1. I can't see informal control being declared voluntarily if it is primarily used to disguise beneficial ownership. Therefore this is primarily about investigative reuse and enhancement of data. In a _policy_ context,  these codes would imply a huge expansion of, and introduction of subjectivity to, BO disclosure requirements (when these could arguably already be covered by catch-all terms like 'other influence or control' in regulations). I think that gives us a challenge in presenting the standard to different groups of stakeholders.
2. There will be a temporal aspect to some of the formal controls granted by financial arrangements, i.e. no control now but may kick in if certain conditions aren't met. This was the thinking behind the `conditional-rights-granted-by-contract` code.

**Comment by siwhitehouse** [comment]
I agree with @ScatteredInk that `loansThinlyCapitalised` is a description of a company, rather than of a person. I also agree that it is a subjective description and that it will be difficult to document how a loan and its provider fall within a scope of declaration. From my naive position I can imagine there would be ways to game the loaning of funds so as to fall outside the scope of a disclosure regime.

As I see it, adding `formal-control-granted-by-provision-of-finance` and `informal-control-granted-by-provision-of-finance` `interest-types` to the codelist would have the following advantages and disadvantages:

Advantages:
- it allows BODS to describe the way in which the control is held - through the provision of finance.

Disadvantages
- it introduces confusion in how to describe the nature of the control. For instance, how would a declaring entity describe the situation were a person has the right to appoint the board through virtue of making a loan to the company?
- this connects how someone gains control to the control they exercise. One danger to this approach is that it has the potential to multiply the number of items in the codelist, make them feel overly specific and add additional overhead to publishers.

In [#327](https://github.com/openownership/data-standard/issues/327) we are exploring the idea of categorising `interest-types` into roles and rights, and I have a bias towards keeping those separate. 

Using that proposal as a guide I can see that we could add a new `role` of `provider-of-finance` which could then have any number of rights (`voting-rights`, `appointment-of-board`, `rights-to-profit-or-income` etc.) associated with it.

#327 is still in discussion. I'm particularly interested in responses that challenge the proposal to categorise `interest-types` as roles and rights.
